openapi/task_execution.swagger.yaml

```diff
@@ -547,14 +526,17 @@ definitions:
           Task logging information.
           Normally, this will contain only one entry, but in the case where
           a task fails and is retried, an entry will be appended to this list.
-        title: OUTPUT ONLY
+        readOnly: true
       creation_time:
         type: string
         description: |-
           Date + time the task was created, in RFC 3339 format.
           This is set by the system, not the client.
-        title: 'OUTPUT ONLY, REQUIRED'
+        readOnly: true
     description: Task describes an instance of a task.
+    required:
+    - executors
+    - creation_time
```

Comment:
OpenAPI 2.0 says parameters marked as `readOnly` should not be in the required list of the schema. So you should remove `creation_time` here.

Reply:
Commit 8f483cec91076eb14fb39e638d6d71df8cf9953d removed `creation_time` from the list of required params for `tesTask`